observations/snow/ghcn_snow.yaml.j2

```diff
@@ -0,0 +1,53 @@
+ -
+
+  # Observation Space (I/O)
+  # -----------------------
+  obs space:
+    name: ghcn_snow
+    obsdatain:
+      engine:
+        type: H5File
+        obsfile: "{{snow_obsdatain_path}}/{{snow_obsdatain_prefix}}{{observation_from_jcb}}.nc"
+    obsdataout:
+      engine:
+        type: H5File
+        obsfile: "{{snow_obsdataout_path}}/{{snow_obsdataout_prefix}}{{observation_from_jcb}}{{snow_obsdataout_suffix}}"
+    simulated variables: [totalSnowDepth]
+  #
+
+  # Observation Operator
+  # --------------------
+  obs operator:
+    name: Identity
+  #
+
+  # Observation Filters (QC)
+  # ------------------------
+  obs filters:
+    - filter: Bounds Check
+      filter variables:
+      - name: totalSnowDepth
+      minvalue: 0.0
+    - filter: Domain Check
+      where:
+      - variable:
+          name: MetaData/height
```

Comment:
I thought we changed to "MetaData/stationElevation" (at least in the offline workflow, not quite sure about g-w)? Therefore, please double-check. Thanks!